ee/packages/ui-theming/src/codeBlockStyles.ts

```diff
@@ -0,0 +1,74 @@
+export const codeBlock = `pre code.hljs {
+	display: block;
+	overflow-x: auto;
+	padding: 1em;
+}
+code.hljs {
+	padding: 3px 5px;
+}
+.hljs {
+	background: #1d1f21 !important !important;
+	color: #c5c8c6 !important;
+}
+.hljs span::selection,
+.hljs::selection {
+	background: #373b41 !important;
+}
+.hljs span::-moz-selection,
+.hljs::-moz-selection {
+	background: #373b41 !important;
+}
+.hljs-name,
+.hljs-title {
+	color: #f0c674 !important;
+}
+.hljs-comment,
+.hljs-meta,
+.hljs-meta .hljs-keyword {
+	color: #707880 !important;
+}
+.hljs-deletion,
+.hljs-link,
+.hljs-literal,
+.hljs-number,
+.hljs-symbol {
+	color: #c66 !important;
+}
+.hljs-addition,
+.hljs-doctag,
+.hljs-regexp,
+.hljs-selector-attr,
+.hljs-selector-pseudo,
+.hljs-string {
+	color: #b5bd68 !important;
+}
+.hljs-attribute,
+.hljs-code,
+.hljs-selector-id {
+	color: #b294bb !important;
+}
+.hljs-bullet,
+.hljs-keyword,
+.hljs-selector-tag,
+.hljs-tag {
+	color: #81a2be !important;
+}
+.hljs-subst,
+.hljs-template-tag,
+.hljs-template-variable,
+.hljs-variable {
+	color: #8abeb7 !important;
+}
+.hljs-built_in,
+.hljs-quote,
+.hljs-section,
+.hljs-selector-class,
+.hljs-type {
+	color: #de935f !important;
```

Comment:
`question` Why so many `!important` declarations?

Reply:
`hljs` injects a css at end of the `head`.
it would be correctly overwritten if the new style was imported on `packages/gazzodown/src/code/CodeBlock.tsx`
but since we need to conditionally toggle between these 2 styles (light theme uses the default, dark theme uses this one), the way out was to "manually" inject it along with our dark palette.

our palette is injected **before** on the dom, that's why `!important` is needed :(